examples/validation-app/src/__tests__/acceptance/validate.acceptance.ts

```diff
@@ -55,6 +55,21 @@ describe('validate properties', () => {
       phoneNum: '999-111-1111',
       capacity: 10,
     };
-    await client.post('/coffee-shops').send(invalidAreaCodeCS).expect(422);
+    await client.post('/coffee-shops').send(invalidAreaCodeCS).expect(400);
+  });
+
+  it('create CoffeeShop failed with invalid type', async () => {
```

Comment:
`fails when creating a CoffeeShop with an invalid type`?

Reply:
will update for the rest of the test case names too.